opcodes/arc-tbl.h

```diff
@@ -571,6 +571,9 @@
 /* addsdw<.f><.cc> 0,limm,limm 0010111011101000F1111111100QQQQQ.  */
 { "addsdw", 0x2EE87F80, 0xFFFF7FE0, ARC_OPCODE_ARC600 | ARC_OPCODE_ARC700, ARITH, NONE, { ZA, LIMM, LIMMdup }, { C_F, C_CC }},
 
+/* add2_jlioff<.f> b,b,s12 00100bbb10010101FBBBssssssSSSSSS.  */
+{ "add2_jlioff", 0x20950000, 0xF8FF0000, ARC_OPCODE_ARC600 | ARC_OPCODE_ARC700 | ARC_OPCODE_ARCv2EM | ARC_OPCODE_ARCv2HS, ARITH, NONE, { RB, RBdup, SIMM12_20_JLIOFF }, { C_F }},
```

Comment:
Then, this mod is not required. We will use the normal add2 instruction plus a relocation. E.g.: add2 rx,ry,@sym@jli. IMHO, it is cleaner in this way.